lib/plug_signature/conn_test.ex

```diff
@@ -222,18 +222,6 @@ defmodule PlugSignature.ConnTest do
     to_string(now + validity)
   end
 
-  defp maybe_add_host_header(%Plug.Conn{host: host} = conn) when is_binary(host) do
-    case get_req_header(conn, "host") do
-      [] ->
-        put_req_header(conn, "host", host)
```

Comment:
That is no longer possible with Plug 1.14, so the whole function does not make sense anymore.